src/app/newSettings/Sections/Account/Billing/BillingAccountSection.tsx

```diff
@@ -28,9 +28,17 @@ const BillingAccountSection = ({ changeSection, onClosePreferences }: BillingAcc
   const [planName, setPlanName] = useState<string>('');
   const [planInfo, setPlanInfo] = useState<string>('');
   const [currentUsage, setCurrentUsage] = useState<number>(-1);
+  const [state, setState] = useState<{ tag: 'ready'; invoices: Invoice[] } | { tag: 'loading' | 'empty' }>({
+    tag: 'loading',
+  });
+  const invoices = state.tag === 'ready' ? state.invoices : [];
 
   useEffect(() => {
     plan.individualSubscription?.type === 'subscription' ? setIsSubscription(true) : setIsSubscription(false);
+    paymentService
+      .getInvoices({ userType: UserType.Individual })
+      .then((invoices) => setState({ tag: 'ready', invoices }))
+      .catch(() => setState({ tag: 'empty' }));
```

Comment:
Shouldn't this only be called when the component is rendered for the first time? Or is it necessary when `individualSubscription` changes? When the `individualSubscription` field changes?